shark/iree_utils/compile_utils.py

```diff
@@ -355,11 +350,16 @@ def export_module_to_mlir_file(module, frontend, directory: str):
 
 
 def get_results(
-    compiled_vm, input, config, frontend="torch", send_to_host=True
+    compiled_vm,
+    function_name,
+    input,
+    config,
+    frontend="torch",
+    send_to_host=True,
 ):
     """Runs a .vmfb file given inputs and config and returns output."""
     device_inputs = [ireert.asdevicearray(config.device, a) for a in input]
-    result = compiled_vm(*device_inputs)
+    result = compiled_vm[function_name](*device_inputs)
```

Comment:
A stylistic ask, if we know that the functions are limited. i.e `compute_loss`, and `step`, would be nice to have it's own function, since it would look better as `vae.step(arg0)` vs vae("step",arg0)`.

Reply:
That's a fair ask and I understand.
Two reasons why, in my opinion, that seems unlikely :-
1. Unlike `forward` (func.func @forward), the accessors generated by IREE is of the following type :-
```
func.func @global$_tensor_constant1$get() -> tensor<f32> {
    %_tensor_constant1 = util.global.load @_tensor_constant1 : tensor<f32>
    return %_tensor_constant1 : tensor<f32>
  }
```
Now invoking the same in Python like, say, `vae.global$_tensor_constant1$()` defeats the "artistic" purpose. :P

2. The number of such functions depends on which model we're dealing with, i.e. how many parameters.